### PR TITLE
[MIG] stock_location_position: Migration to 14.0

### PR DESCRIPTION
**Pull request by phuctranfxvn**

### CODE REVIEW

**Review by dreispt — APPROVED** [review]

File: stock_location_position/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    "name": "Stock Location Position",
+    "version": "14.0.1.0.0",
+    "author": "BCIM, Okia, Camptocamp, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/stock-logistics-warehouse",
+    "summary": "Add coordinate attributes on stock location.",
+    "category": "Stock Management",
+    "depends": ["stock"],
+    "data": ["views/stock_location.xml"],
+    "installable": True,
+    "development_status": "Alpha",
```
Comment (rousseldenis) [inline stock_location_position/__manifest__.py:15]: Is this still Alpha ?

**Review by rousseldenis — APPROVED** [review]
Code review. Just one remark

File: stock_location_position/__manifest__.py (hunk 1 above)
Comment (phuctranfxvn) [reply stock_location_position/__manifest__.py:15]: Hello @rousseldenis 

I am not sure the status of this module. Maybe we need to do more test to evaluate the module as stable ? Let me know

Thanks

**Pull request merged**